template/before.sh.erb

```diff
@@ -8,7 +8,7 @@
           when "2.3.0"
             "py4j-0.10.6-src.zip"
           when "3.0.1"
-            "py4j-0.10.9-src.zip"
+            ENV['SPARK_PY4J_PATH']
```

Comment:
environment variables won't work here. This ERB portion is evaluated when the script is submitted (before the job starts) on the web server, which doesn't have modules.

So this environment variable will evaluate to nil because the modules haven't been loaded yet.

Reply:
Thanks for pointing that out. Made the change to remove the entire case statement and use the env var set after the module is loaded.